feed/serializers/atom_extensions.py

```diff
@@ -28,16 +28,16 @@ def extend_atom(self: BaseExtension, atom_feed: Element) -> Element:
             The feed's root element.
 
         """
-        # Remove the unwanted "generator" element, which can't be erased though the API.
+        # Remove the unwanted "generator" element, which can't be erased though
+        # the API.
         for child in atom_feed:
-            if child.tag == 'generator':
+            if child.tag == "generator":
                 atom_feed.remove(child)
```

Comment:
As of feedgen 0.9.0 there's no longer a need to do this, you can do `fg.generator("")` after instantiating the feed; see https://github.com/arXiv/arxiv-search/blob/65ff0520c44f616908da141812df7b25ae195b5e/search/serialize/atom.py#L146

Reply:
Done.